cypher/cypher-docs/src/docs/dev/ql/start/index.asciidoc

```diff
@@ -4,31 +4,42 @@
 [abstract]
 Find starting points through explicit indexes.
 
-[IMPORTANT]
-The `START` clause was removed in Cypher 3.2.
-Using the `START` clause will cause the query to fall back to using Cypher 3.1.
-`START` should only be used when accessing legacy indexes.
-In all other cases, use `MATCH` instead (see <<query-match>>).
-
-* <<start-node-by-index-seek, Node by index seek>>
-* <<start-relationship-by-index-seek, Relationship by index seek>>
-* <<start-node-by-index-query, Node by index query>>
-
-In Cypher, every query describes a pattern, and in that pattern one can have multiple starting points.
-A starting point is a relationship or a node where a pattern is anchored.
-Using `START` you can only introduce starting points by explicit index seeks.
-Note that trying to use a explicit index that doesn't exist will generate an error.
-
-This is the graph the examples are using:
-
-.Graph
-include::includes/cypher-start-graph.asciidoc[]
-
-== Get node or relationship from index
-
-include::node-by-index-seek.asciidoc[leveloffset=+1]
-
-include::relationship-by-index-seek.asciidoc[leveloffset=+1]
-
-include::node-by-index-query.asciidoc[leveloffset=+1]
+The `START` clause was removed in Cypher 3.2, and the recommendation is to use `MATCH` instead (see <<query-match>>).
+However, if the use of explicit indexes is required, a series of built-in procedures, detailed below, allows these to be managed and used.
+These procedures offer the same functionality as the `START` clause.
+In addition, queries using these procedures will exhibit superior execution performance over queries using `START`.
```

Comment:
They 'may' exhibit superior performance due to the cost planner. But it is not for certain, many simple queries performed the same between RULE and COST. I would word it:

> In addition, queries using these procedures may exhibit superior execution performance over queries using `START` due to the use of the `COST` planner and newer Cypher 3.2 compiler.